### ISSUE TITLE
Crash on startup on fenix5x

### ISSUE THREAD

**Issue by warmsound** [warmsound/crystal-face#16]
Reported by Karsten Leven. Shows IQ symbol when selecting watch face.
Version 1.5.1, SDK 2.4.2.

[CIQ_LOG.TXT](https://github.com/warmsound/crystal-face/files/1882110/CIQ_LOG.TXT)
[Crystal.debug.xml.zip](https://github.com/warmsound/crystal-face/files/1882112/Crystal.debug.xml.zip)

**Comment by warmsound** [comment]
Callstack determined as:
```
GoalMeter.mc: 356 (function getSegmentScale())
GoalMeter.mc: 283 (function getSegments())
GoalMeter.mc: 74 (function setValues(current, max))
CrystalView.mc: 396 (function updateGoalMeter(goalType, meter, iconLabel, currentLabel, maxLabel))
CrystalView.mc: 383 (function updateGoalMeters())
```

Likely cause is that right goal meter's max value is 0. On fenix5x, this is `floorsClimbedGoal`. This can be reproduced in the simulator by setting e.g. Floors Climbed Goal or Weekly Active Minutes Goal to zero.

**Comment by warmsound** [comment]
Fixed on master in commit dec1720.

Note that, on current version of Garmin Connect on iPhone, only the Floors Climbed goal can be set to 0; steps and active minutes cannot be.
